cleverhans/model.py

```diff
@@ -105,11 +86,8 @@ def __init__(self, callable_fn, output_layer):
         self.output_layer = output_layer
         self.callable_fn = callable_fn
 
-    def get_layer_names(self):
```

Comment:
Why remove get_layer_names?

Reply:
It's not used anywhere as far as I can tell, it seems it was something inherited from Keras.
I can put it back in for backward compatibility, but I don't know who would use this if it's not even used in the CleverHans code. What's your take on this?

Reply:
Ok done putting it back.